configure.patch

```diff
@@ -0,0 +1,26 @@
+--- a/configure	2022-10-10 02:38:18.000000000 -0500
++++ b/configure	2024-05-13 17:23:16.686952633 -0500
+@@ -211,6 +211,7 @@
+         /usr/lib/x86_64-linux-gnu
+         /usr/lib/i386-linux-gnu
+         /usr/lib/arm-linux-gnueabihf
++        /usr/lib/aarch64-linux-gnu
+     }
+ 
+     if {! [info exists var(TCL_INCLUDE)]} {
+@@ -490,13 +491,8 @@
+     if {[isDarwin]} {
+         set var(FONTDIR) /Library/Fonts/
+     } else {
+-        # Just install fonts in to /usr irrespective of system prefix. /usr/local may not be active
+-        set prefix /usr
+-        if {![file isdirectory $prefix/share/fonts]} {
+-            set var(FONTDIR) "~/.fonts"
+-        } else {
+-            set var(FONTDIR) $prefix/share/fonts/truetype/Scid
+-        }
++        set var(FONTDIR) "~/.fonts"
```

Comment:
This should be set to `~/.local/share/fonts`. This location has been deprecated since a very long time and Flatpak will not mount fonts from both `~/.fonts` and `~/.local/share/fonts`, it will mount from only one.

Reply:
Also how will this work? It has no access to `~/.fonts` right now and flatpak mounts fonts from `~/.fonts` or `~/.local/share/fonts` into `/run/host`.

Reply:
The part you mention, I forgot to delete it. If I'm not wrong, it doesn't prevent the compilation of the package.

I have also not seen the package behave inappropriately.

(this is a translation)